package.json

```diff
@@ -37,7 +37,7 @@
     "prepare": "install-peers",
     "test": "run-s test:circular lint format:check",
     "test:circular": "dpdm --warning false --tree false --exit-code circular:1 ./src/index.ts",
-    "lint": "eslint . --max-warnings=203 --color",
+    "lint": "eslint . --max-warnings=199 --color",
```

Comment:
We have a winner :1st_place_medal:

Reply:
Bravo @jgyselov !

Reply:
Good team effort! Bravo to all of us! :fireworks: